### PR TITLE
feat(Poker): Add a timer to Sprint Poker

### LINKED ISSUE #6843: Add a timer to Sprint Poker
## Issue - Enhancement
A user requests:
I want a timer in Planning Poker. The same one as retros have.

Seems reasonable and relatively simple, given the pattern already exists?

### PR DESCRIPTION
**Pull request by 87prashant**
# Description

Close #6843

## Demo
Screenshot 2022-07-03 121815
Screenshot 2022-07-03 121704

## Testing scenarios

- Start a new Poker meeting
- Verify the timer in different Phases

## Final checklist

- [X] I checked the [code review guidelines](../docs/codeReview.md)
- [X] I have performed a self-review of my code, the same way I'd do it for any other team member
- [X] I have tested all cases I listed in the testing scenarios and I haven't found any issues or regressions
- [X] Whenever I took a non-obvious choice I added a comment explaining why I did it this way
- [ ] I added the label `One Review Required` if the PR introduces only minor changes, does not contain any architectural changes or does not introduce any new patterns and I think one review is sufficient'
- [X] PR title is human readable and could be used in changelog

### CODE REVIEW

File: packages/client/components/MeetingControlBar.tsx [hunk 1]
```diff
@@ -200,7 +200,8 @@ const MeetingControlBar = (props: Props) => {
                 <StageTimerControl
                   {...tranProps}
                   cancelConfirm={cancelConfirm}
-                  defaultTimeLimit={DEFAULT_TIME_LIMIT[phaseType]}
+                  //default time limit of Poker is 3
+                  defaultTimeLimit={DEFAULT_TIME_LIMIT[phaseType] || 3}
```
Comment (nickoferrall) [inline packages/client/components/MeetingControlBar.tsx:204]: -1 is there a way that we can accomplish this without hardcoding the poker time limit?

**Review by nickoferrall — CHANGES_REQUESTED** [review]
Thanks for the PR! I'm excited to get this one fixed

File: packages/client/components/MeetingControlBar.tsx (hunk 1 above)
Comment (87prashant) [reply packages/client/components/MeetingControlBar.tsx:204]: Thanks for the suggestion, working on it

Comment (87prashant) [reply packages/client/components/MeetingControlBar.tsx:204]: I am using Scope: 3 and Estimate: 5.
Is that ok?

Comment (87prashant) [reply packages/client/components/MeetingControlBar.tsx]: @nickoferrall 
Polite reminder

**Review by nickoferrall — APPROVED** [review]
LGTM!

**Comment by mattkrick** [comment]
The timer doesn't show up in the estimate phase. this will need design work. removing the PR for the ship, we can fix this for the next release.

**Pull request merged**